lib/src/main/java/ca/jbrains/pos/domain/Price.java

```diff
@@ -0,0 +1,13 @@
+package ca.jbrains.pos.domain;
+
+public class Price {
```

Comment:
Interesting! The class is called `Price`, but you also have the more-generic words "money" and "amount". :thinking: :wink:

Reply:
yeah, I find Price would reflect the domain better, although one possibility that I could see is we could make Price compose of Money

Reply:
Indeed, `Price` is good enough, but eventually this will probably become something more general, like `MonetaryAmount`. Now you can choose when to generalize the name: now, while you're thinking of it, or later, when you'll obviously want to do it. For a refactoring this small, I think both options work equally well.